swagger.yml

```diff
@@ -373,6 +373,55 @@ paths:
             $ref: '#/definitions/envVar'
         default:
           $ref: '#/responses/error'
+    patch:
+      tags:
+        - environmentVariables
+      operationId: setEnvVarValue
```

Comment:
Hmm, it seems to be ordered by number of path segments, ascending: `/accounts/{account_id}/env` is first, and `/accounts/{account_id}/env/{key}/value/{id}` is last. My preference would by to group by HTTP verb: e.g. the GETs would be grouped together within the sidebar (and on the page). I wonder if there's a configuration option of this site's frameword that would allow that.

Reply:
Looks like `sortOperationsAlphabetically` is [an option](https://github.com/Redocly/redoc#redoc-options-object) -- I created an issue: https://github.com/netlify/open-api/issues/388